wolfcrypt/test/test.c

```diff
@@ -12856,7 +12856,7 @@ static int rsa_sig_test(RsaKey* key, word32 keyLen, int modLen, WC_RNG* rng)
     /* RNG is handled with the cryptocell */
     if (ret != 0)
 #else
-    if (ret != MISSING_RNG_E)
+    if (ret == MISSING_RNG_E)
```

Comment:
This test is expected to fail because the RNG is missing. And it should return `MISSING_RNG_E`. What are you seeing on the ESP32 for the return code here?

Reply:
The "error" occurs here:
![image](https://user-images.githubusercontent.com/13059545/181772418-70433924-9550-43c7-a4fe-21f04206288b.png)

causing this to be skipped:
![image](https://user-images.githubusercontent.com/13059545/181772872-651d603b-ffb8-4458-9705-c546f5745b51.png)

when that line does not return an error, I see this:
![image](https://user-images.githubusercontent.com/13059545/181777141-f7b995f1-b69f-4e08-8f16-ee9c7edabc8c.png)

I should have probably trusted my first instinct: the watchdog is likely starved, even though I have a high timeout value:
![image](https://user-images.githubusercontent.com/13059545/181777455-59f1b034-813d-40ce-bc51-596acf33ca15.png)

When I saw the `MISSING_RNG_E` I thought it was hardware-based problem, since I'm venturing into new territory.

Given your comment that it is _supposed_ to be an error, I will try feeding the watchdog and close this PR, agreed? That RSA step, particularly without HW acceleration, is pretty time consuming.

Reply:
All that looks fine to me. The RSA math here takes a while, and it looks like your watchdog is triggering. Is it possible you need a vTaskDelay(0) introduced between the tests? See test.c `TEST_SLEEP_MS`. Otherwise could be a stack space issue. I think you should close this PR and work on the root issue.

Reply:
ok, cool. thanks for the tip on `TEST_SLEEP_MS`, I'll check it out.